ruby/spec/cucumber/ci_environment/ci_environment_spec.rb

```diff
@@ -8,7 +8,12 @@
     context "with #{File.basename(test_data_file, '.txt')}" do
       subject { JSON.parse(ci_environment.to_json) }
 
-      let(:ci_environment) { Cucumber::CiEnvironment.detect_ci_environment(env) }
+      def mockReader(path)
```

Comment:
```suggestion
      def mock_reader(path)
```

(This kind of casing is used in the rest of the Ruby code.)